doc/specifications/proposed/named_queries/named_queries.md

```diff
@@ -12,6 +12,15 @@ ContentType identifier...
 In the initial version, QueryType objects can be created as PHP classes. The main use-case is to allow developers
 to define Queries for the project that can be used in several ways, without knowing their internals.
 
+### Differnces from Persisted Queries
+
+When `Views` where added to REST API there was the intention to allow these to be persisted at some point (after UI where moved to Platform Stack). And while there are no own specification for it yet, these are the main differences between the two:
+- Persisted Queries serves as Editorial Queries/Search for future possible features like reuse in content, blocks, and plainly sharing link to a view to co workers or public.
+- Named Queries on the other side is a DX feature for devloper provided queries for reuse in code, and aim to make eZ Platfrom much easier to work with by exposing the power of `filtering` views direclty from templates.  
+- Persisted Queries, unlike Named Queries, will by it's nature be possible to be retrive and created from PHP Repository API, and hence also REST API which will need distingusing these two from eachother for UI's exposing such  editing functionaity.
+- Persisted Queries has the possibility to allow Repository to keep track of hits internally in the future, ala how JIRA works with filters. This will allow more advance cache clearing logic to aim for editor never having to take care about cache.
```

Comment:
What if a Filter (it's a good name) was (transparently) _based_on a query, that itself might have parameters? We could keep track of it, and creating the filter could involve defining whatever parameters are required by it (this would involve an programmatic definition of the parameters, with type and everything).